engine/apps/api/urls.py

```diff
@@ -106,5 +108,3 @@
     path(r"login/<backend>/", auth.overridden_login_slack_auth, name="slack-auth"),
     path(r"complete/<backend>/", auth.overridden_complete_slack_auth, name="complete-slack-auth"),
 ]
-
-urlpatterns += router.urls
```

Comment:
this line is essentially a duplicate of [line 69](https://github.com/grafana/oncall/pull/2611/files#diff-d87ee8df9c3c7eed2c4b729461f3831c5318b69907c58cdc0a2e566c170c2e9bR69) and can be removed:
![Screenshot 2023-07-21 at 15 49 00](https://github.com/grafana/oncall/assets/9406895/991cf4c9-05f1-4c3b-abe2-af0d8fd72822)